routes/truckRoutes.js

```diff
@@ -1,3 +1,118 @@
-// connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
+// connect to mongodb in this module as this is where you'll be making create/read/delete calls to your database
 // use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
 // remember this is a Node module
+var express = require('express');
+var app = express.router();
+
+// mongoose is just another node module
+var mongoose = require('mongoose');
+var db = mongoose.connect('mongodb://localhost/trucksAPI');
+var bodyParser = require('body-parser');

```

Comment:
You've already used the bodyParser middle-ware in app.js, so you don't need to use it again inside truckRoutes.js